docs/modules/ROOT/pages/getting-started/cloud-trial/Signup Payara Cloud.adoc

```diff
@@ -1,33 +1,72 @@
 = Sign Up To Payara Cloud
 
-Payara Cloud offers 15 days free trial.
+Payara Cloud offers a 15-day free trial to explore the service without commitment.
+To get started, head over to the https://login.payara.cloud/u/signup[signup page] and fill in the signup form.
 
-Create an account https://billing.payara.cloud/signup/plans[here].
+.Signup Form
+image::new-signup-1.png[Payara Cloud Signup Form, text Description automatically generated,width=672,height=361, link="{imagesdir}/new-signup-1.png", window="_blank"]
 
-Choose the Free Trial option and the Plan you would like your subscription to transition to, if you choose to continue after the 15 day trial is over. If you choose not to continue using Payara Cloud, you will have the opportunity to quickly and easily xref:getting-started/cloud-trial/Cancel Payara Cloud.adoc[Cancel Subscription].
+.Signup Form
+[checklist]
+* ❶ Enter a username.
+* ❷ Enter your email.
+* ❸ Enter a strong password that is at least 8 characters long, has a number, uppercase, lowercase and/or special character
+* ❹ Click to continue
+* ❺ Alternatively you can sign up with your Google account
+* ❻ Or your GitHub Account
 
-image::cloud-trial-image1.png[image,width=724,height=387, link="{imagesdir}/cloud-trial-image1.png", window="_blank"]
 
-Click btn:[Continue to Invoicing].
+NOTE: Ensure you use a valid email address for important account notifications.
+
+After clicking Continue, an account confirmation email will be sent to the email address you entered in the signup form.
+
+Click the btn:[Verify Your Account] button in the email to successfully verify your email.
+You can now proceed to https://login.payara.cloud/u/login[log into] your new Payara Cloud account.
+
+
+== Create A Subscription
+To use Payara Cloud for the first time after logging into your account, you must create a subscription.
+A Payara Cloud subscription represents the billable unit for your usage of the Payara Cloud service.
+
+.Blank subscription page
+image::new-signup-2.png[Payara Cloud Empty Subscription Page, text Description automatically generated,width=672,height=361, link="{imagesdir}/new-signup-1.png", window="_blank"]
+
+Click on the btn:[Sign-Up To Payara Cloud] button to navigate to the subscription creation page.
+
+.Subscription Creation Page
+image::new-signup-3.png[Payara Cloud New Subscription Page, text Description automatically generated,width=672,height=361, link="{imagesdir}/new-signup-3.png", window="_blank"]
+
+On the subscription creation page, leave the default selection on the free trial and click on the btn:[Continue To Invoicing] button.
+
+.Invoicing Tab
+image::new-signup-4.png[Payara Cloud Invoicing Info, text Description automatically generated,width=672,height=361, link="{imagesdir}/new-signup-4.png", window="_blank"]
+
+On the ensuing invoicing tab, set the name of the primary contact for the account and click the btn:[Continue To Confirmation] button.
```

Comment:
```suggestion
On the ensuing customer details tab, set the name of the primary contact for the account and click the btn:[Continue To Confirmation] button.
```

Reply:
I'll leave it as invoicing as that is what the UI shows. using "customer details" can be confusing because the UI shows it as invoicing.
![image](https://github.com/payara/Payara-Cloud-Documentation/assets/27694244/6bb8d046-6636-4d7f-9f6a-d6249d769c24)
![image](https://github.com/payara/Payara-Cloud-Documentation/assets/27694244/86f31bcb-1301-4be2-aebd-d9879001f20f)

Reply:
I'll file an improvement ticket to make the naming consistent.

Reply:
yes, thank you